content/foundation/governing-board/index.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Governing Board"
+weight = 3
+extra.summary = """
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team.
+"""
++++
+
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team. It is organized under [these bylaws](/media/2024-04-governing-board-terms-of-reference.pdf) (v1.2, last updated April 2024).
+
+There are nine different constituency groups that are allocated seats on the Governing Board, across three categories:
+
+* **Community representatives**
+  * 4 seats for Individual Members — these are people who currently provide financial support to the Foundation.
+  * 3 seats for Ecosystem Members — these are FOSS projects and communities around Matrix.
+  * 2 seats for Associate Members — these are FOSS foundations, academic institutions, and other communities that are aligned with key parts of our mission.
+* **Funder representatives**
+  * 4 seats for Platinum Members — these organisations sponsor at the highest level.
+  * 3 seats for Gold Members — these organisations sponsor at a high level.
+  * 2 seats for Silver Members — these organisations sponsor at a level commensurate with the size of their organisation.
+* **Foundation representatives**
+  * 3 seats for [Guardians](/foundation/about#the-guardians) — these are the members of the Foundation's board of directors.
+  * 2 seats for [the Spec Core Team](/foundation/about#the-spec-core-team) — these are the members of the Spec Core Team that looks after the spec.
+  * [Managing Director](/foundation/about#managing-director) — this is the Foundation's head of staff.
+
+Elections are being held in the second quarter of each calendar year, alternating halves of the Constituency Groups representatives on the Governing Board as per the following calendar:
+
+* April-June 2024, and any subsequent even-numbered years: Guardians, SCT, Platinum and Ecosystem
+* April-June 2025, and any subsequent odd-numbered years: Gold, Silver, Individual, Associate
+
+## Elected representatives
```

Comment:
I understand why it's been done like this, but I think it would be preferable to still name Robin as a GB member or it will we too easy to glance over the fact that the MD is on the board

Reply:
Agreed, add a line perhaps after "Guardians" for this